src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -43,25 +43,38 @@
   
   <PropertyGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <DefineConstants>FUNCTIONS_V1</DefineConstants>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!--Don't include Correlation classes in Function V1.-->
+    <Compile Include="**/*.cs" Exclude="Correlation/*.cs;**/obj/**/*.cs" Condition="'$(TargetFramework)' != 'netstandard2.0'" />
+  </ItemGroup>
+
   <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.3.0" />
 
     <!--Don't increase below versions without significantly testing on Functions V1. Increasing these versions increments
     some dependencies that have binding redirects in Functions V1.-->
     <PackageReference Include="Microsoft.Extensions.Options" Version="1.0.2" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="1.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Routing" Version="1.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.WebApiCompatShim" Version="1.0.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Routing" Version="1.0.5" /> <!-- used to be 1.0.3 -->
```

Comment:
Not sure why this was changed... it concerns me a bit, as Functions v1 is fairly fragile with this.

Reply:
@TsuyoshiUshio do you know why this changed?

Reply:
I don't clearly remember it. However, Chris said that this feature (Distributed Tracing) is only for v2. (to encourage people to move v2) So that I try to make it.  It is just a matter of the compile of the library.

Reply:
This has been simplified in the my last few commits